test/box-luatest/gh_7894_export_iproto_constants_and_features_test.lua

```diff
@@ -3,6 +3,8 @@ local t = require('luatest')
 
 local g = t.group()
 
+local is_enterprise = require('tarantool').package == 'Tarantool Enterprise'
```

Comment:
Nit: Better use the luatest helper `t.tarantool.is_enterprise_package()`.

Reply:
Done